src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.report;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+
+    @Override
+    public String getReport(Map<String, Integer> inventory) {
+        // Додаємо заголовок
+        StringBuilder report = new StringBuilder(
+                "fruit,quantity\n");
```

Comment:
move all literals to the constant fields. Also better use System.lineseparator()